sql/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -561,4 +561,44 @@ public void testJoinWithIndexMissingExceptions() throws Throwable {
             throw Exceptions.unwrap(t);
         }
     }
+
+    private void setupLeftJoinTestData() {
```

Comment:
imho it is easier to read with t1, t2 etc so i would rather use those also because there is already another schema in this class. however not so important.